builtin/credential/approle/validation.go

```diff
@@ -152,7 +138,7 @@ func (b *backend) validateCredentials(ctx context.Context, req *logical.Request,
 		}
 	}
 
-	return role, roleName, metadata, secretID, nil, nil
+	return role, roleIDIndex.Name, metadata, secretID, nil, nil
```

Comment:
Why do we return `secretID`? The only caller doesn't even use it.

Reply:
I removed the function as it had a single caller.